client/src/App.js

```diff
@@ -15,7 +16,11 @@ function App() {
   const [userData, setUserData] = useState({
     token: undefined,
     user: undefined,
+    polls: undefined,
+    followers: undefined,
+    followings: undefined,
```

Comment:
Since these states are starting to grow, you may want to separate them out to its own individual state, and pass it in individually through the context provider. Because you probably won't need _all_ of this info at the same time in all the components